SystemConnectors/WindowsConnector/ProcessController.py

```diff
@@ -13,6 +13,8 @@ def get_running_processes(self):
         sessions = AudioUtilities.GetAllSessions()
         process_names = []
         for session in sessions:
-            if session.Process and session.Process.name:
-                process_names.append(session.Process.name())
+            if session.Process and session.Process.name():
+                name = session.Process.name()
+                cleanName = name[0:name.rfind(".")]
+                process_names.append(cleanName != -1 if cleanName else name)
```

Comment:
![image](https://user-images.githubusercontent.com/16418232/102001463-93715e80-3cc0-11eb-8b49-c85679b3aef3.png)

I don't think this is doing what you think it is